lib/src/union_set_controller.dart

```diff
@@ -23,8 +23,8 @@ import 'union_set.dart';
 /// ```
 class UnionSetController<E> {
   /// The [UnionSet] that provides a view of the union of sets in [this].
-  UnionSet<E> get set => _set;
-  UnionSet<E> _set;
+  UnionSet<E>? get set => _set;
+  UnionSet<E>? _set;
```

Comment:
We don't want these to be nullable. Maybe this should be `late`?

Can we move the assignment to an initializer list and make if `final` and non-nullable?

Reply:
I made it late final - I can't move the assignment to the initializer list because it requires another field.

I could make a factory and private constructor if you think that would be better though...

Reply:
factory + private constructor would, I think, be technically breaking if someone `extends UnionSet`, but we maybe can ignore that. Let's file an issue and follow up separately.

Reply:
https://github.com/dart-lang/collection/issues/124